src/main/java/carpet/helpers/HopperCounter.java

```diff
@@ -154,4 +157,44 @@ public long getTotalItems()
     {
         return counter.values().stream().mapToLong(Long::longValue).sum();
     }
-}
+
+    private static String getPrettyColour(DyeColor colour){
+        return getPrettyColour(colour, "");
+    }
+
+    private static String getPrettyColour(DyeColor colour, String extra){
+        if (MaterialColor.WHITE.equals(colour.getMaterialColor())) {
```

Comment:
This should be defined in a map, specifying appropriate formatting string given a colour.

Reply:
And that mapping, as generic, could even be specified in WoolTool (I could then rename it to ColourTool)